src/ZODB/scripts/migrate.py

```diff
@@ -85,7 +85,7 @@
 from ZODB.TimeStamp import TimeStamp
 
 PROGRAM = sys.argv[0]
-ZERO = '\0'*8
+ZERO = b'\0' * 8
```

Comment:
```py
ZERO = utils.z64
```

Reply:
or replace the only use of `ZERO` with `utils.z64`

Reply:
Thanks for the suggestions! I was trying to keep the changes as minimal as possible, but since you brought it up I'm more than happy to DRY.

Reply:
This script doesn't even run or import because there  is no `from ZODB import StorageTypes`